src/SessionManager/ProgramSettings.cc

```diff
@@ -137,6 +137,7 @@ void ProgramSettings::ApplyCommandLineSettings(int argc, char** argv) {
         ("log_protocol_messages", "enable protocol message debug logs", cxxopts::value<bool>())
         ("no_http", "disable frontend HTTP server", cxxopts::value<bool>())
         ("no_browser", "don't open the frontend URL in a browser on startup", cxxopts::value<bool>())
+        ("browser", "user custom browser + args", cxxopts::value<string>(), "<browser>")
```

Comment:
shouldn't it be `"use custom browser + args"`? I prefer something along the lines of 

```
Specify command to use to invoke a web browser when opening the CARTA frontend. If not specified, the default browser is used.
```

Reply:
We should not be putting long explanations in the command descriptions. Any long explanation should go in the body of the help.

Reply:
fair enough. But `user -> use` makes sense, right?

Reply:
can we have _one_ example for args so that users can be inspired for the alternative options?

Reply:
I suggest  `custom browser command` here, with an explanation later; something like:

`Disabling the browser takes precedence over a custom browser command. The custom browser command may contain the placeholder CARTA_URL, which will be replaced by the frontend URL. If the placeholder is omitted, the URL will be appended to the end.`

(Or whatever we use for the placeholder, which can't be `%url`, as I explained in the other comment.)

Reply:
> can we have _one_ example for args so that users can be inspired for the alternative options?

I would really prefer not to put that in the executable help. It's much too specific for this context. I do think that we should add some information about this to the _manual_, and we can give lots of examples there.